docs/api-ref/rest-api/index.md

```diff
@@ -16,6 +16,60 @@ Prefect Cloud and a locally hosted Prefect server each provide a REST API.
 - Interactive REST API documentation for a locally hosted open-source Prefect server is available at <a href="http://localhost:4200/docs" target="_blank">http://localhost:4200/docs</a> or the `/docs` endpoint of the [`PREFECT_API_URL`](/concepts/settings/#prefect_api_url) you have configured to access the server. You must have the server running with `prefect server start` to access the interactive documentation.
 - The REST API documentation for a locally hosted open-source Prefect server is also available in the [Prefect REST API Reference](/api-ref/rest-api-reference/).
 
+## Interacting with the REST API
+
+You have many options to interact with Prefect REST API
+- create an instance of [`PrefectClient`](/api-ref/prefect/client/orchestration/#prefect.client.orchestration.PrefectClient) 
+- use your favorite Python HTTP library such as [requests](https://requests.readthedocs.io/en/latest/) or [httpx](https://www.python-httpx.org/)
+- use an HTTP library in your language of choice
+- use [CURL](https://curl.se/) from the command line 
```

Comment:
This isn't showing up as bullet points in the preview. Not sure if broken preview:
https://deploy-preview-9661--prefect-docs-preview.netlify.app/api-ref/rest-api/#interacting-with-the-rest-api